Book-A-Desk.Api/Program.fs

```diff
@@ -10,28 +10,25 @@ open Microsoft.Extensions.Configuration
 open Microsoft.Extensions.Hosting
 open Microsoft.Extensions.DependencyInjection
 open Giraffe
-open System.Threading.Tasks
 
 open Book_A_Desk.Api
-open Book_A_Desk.Domain
 open Book_A_Desk.Domain.Office.Domain
 open Book_A_Desk.Infrastructure
-open Book_A_Desk.Infrastructure.DynamoDbEventStore
 open Book_A_Desk.Domain.CommandHandler
  
 let configureCors (ctx : WebHostBuilderContext) (builder : CorsPolicyBuilder) =
-    if ctx.HostingEnvironment.IsDevelopment() then
+//    if ctx.HostingEnvironment.IsDevelopment() then
```

Comment:
You might want to remove these before merging :)

Reply:
Done